### PR TITLE
added home-screen features

### LINKED ISSUE #119: Create the application home screen (with button to make new ATR)
Blocked by #129, #127, #134, and #133.

- [ ] Add a button or something to `NewScreen` to start a new ATR
> The existing ATR editing screen takes in an ATR. Give it a "blank" ATR, or the cached/default ATR.
- [ ] Update `NewScreen` to look like our [prototype](https://xd.adobe.com/view/9a6f1910-d3ac-4fdf-b051-c2c78510ba32-cf3c/screen/737037be-20e5-4d30-a0c0-66dc2c7a2edf)

### LINKED ISSUE #158: Create the Sign out button
Blocked by #153 and #152 .

- [ ] Add the option to sign out as per our Hi-Fi design

You will need to talk to `AuthenticationService`/`FirebaseAuth` in order to sign out the current user. Talk to @MummenRider who has done some work like this already in #95 .

### PR DESCRIPTION
**Pull request by mansiPaTeLL**
@holtzmak added some features of the home screen. The image feature is currently is static as it requires more work. This is a static UI screen. I also need to include navigation for the active screen and history screen. I need to know what model should I use to navigate to the active and history screen. Right now, I have temporarily added a logout button for testing.

### CODE REVIEW

**Comment by mansiPaTeLL** [comment]
image

**Review by MummenRider — APPROVED** [review]
LGTM, I'll probably wait for this commit to be pushed before I merge #193

File: app/lib/ui/views/new/new_screen.dart
Comment (holtzmak) [inline app/lib/ui/views/new/new_screen.dart:21]: Do we need a `stack` in a `column`? Can we drop the column?

File: app/lib/ui/views/new/new_screen.dart
Comment (holtzmak) [inline app/lib/ui/views/new/new_screen.dart:118]: Lmao, all the buttons start a new ATR huh.

File: app/lib/ui/views/new/new_screen.dart
Comment (holtzmak) [inline app/lib/ui/views/new/new_screen.dart]: Can we say "Sign out" here? We call it sign out throughout the application

**Review by holtzmak — APPROVED** [review]
@mansiPaTeLL LGTM! Orange is fun, obviously will change later. 

You can use the `NewScreenViewModel` and give it functions `navigateToActiveScreen`, etc. However, that is blocked by changes to navigator which I will be doing straight away. You can merge this as-is.

File: app/lib/ui/views/new/new_screen.dart
Comment (mansiPaTeLL) [reply app/lib/ui/views/new/new_screen.dart:118]: @holtzmak yea, because I don't know which model should I use for active and history. Did you use `model.HistoryScreenViewModel` for history and `mode.saveEditedAtr` or `mode.saveCompletedAtr` for active screen?

File: app/lib/ui/views/new/new_screen.dart
Comment (mansiPaTeLL) [reply app/lib/ui/views/new/new_screen.dart]: > @mansiPaTeLL LGTM! Orange is fun, obviously will change later.
> 
> You can use the `NewScreenViewModel` and give it functions `navigateToActiveScreen`, etc. However, that is blocked by changes to navigator which I will be doing straight away. You can merge this as-is.

yes, orange is fun! I bored with green lol !!

File: app/lib/ui/views/new/new_screen.dart
Comment (holtzmak) [reply app/lib/ui/views/new/new_screen.dart:118]: I have the fix for this in my latest. I'll handle it

**Pull request merged**
